### DISCUSSION TITLE
Disable adding repositories

### DISCUSSION THREAD

**Discussion by rleffd** [artifacthub/hub#4122] [Q&A] [answered]
Hello,

Is there a way to disable users to create organizations and repositories in their personnal account ?

I'd like to allow only a few users to add repositories in one organization.

I know that it is possible to restrict the access to an organization using the policies (rbac), but how can I restrict users  that are not members of an organization to add repositories using their personal account ?

Thanks

**Comment by tegioz** [comment, marked as answer]
Hi @rleffd 👋 

No, that's not possible. I think the closest you could get is to [not allow users to sign up](https://artifacthub.io/packages/helm/artifact-hub/artifact-hub?modal=values-schema&path=hub.server.allowUserSignUp), as only registered users can publish content on an Artifact Hub instance. The problem is that they'd lose some features, like subscribing to packages notifications or the ability to star packages, for example.

**Reply by rleffd** [reply]
All users are signing with oidc. I have already disbale the signup feature.

Thanks for your reply.
